docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -1,52 +1,73 @@
 # How to Setup a small Dev Cluster of multi-nodes
 
-It may be desired to setup a dev cluster having 2 or 3 nodes in order to play with comprehansive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster havinbg 2 nodes (1 master, plus a worker who joins later).
+It may be desired to setup a dev cluster having more worker nodes in order to play with comprehensive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster having 1 master node and extra worker nodes joining after the master is up.
```

Comment:
Seems to me even if we want just one worker node, we need to follow the instructions here. If that is the case, I think we should say "setup a dev cluster having one or more worker nodes".

Reply:
will change as suggested